ticketview/src/main/java/com/vipulasri/ticketview/TicketView.java

```diff
@@ -602,6 +603,15 @@ private void setShadowBlurRadius(float elevation) {
         mShadowBlurRadius = Math.min(25f * (elevation / maxElevation), 25f);
     }
 
+    public int getShadowColor() {
+        return mShadowColor;
+    }
+
+    public void setShadowColor(int dividerColor) {
```

Comment:
can you please change the param name so that the method can appropriately indicate what's it expecting?

Reply:
Yes for sure!